spec/decorators/event_decorator_spec.rb

```diff
@@ -1,6 +1,6 @@
 # encoding: utf-8
 
-#  Copyright (c) 2012-2013, Jungwacht Blauring Schweiz. This file is part of
+#  Copyright (c) 2012-2021, Jungwacht Blauring Schweiz. This file is part of
```

Comment:
Die Änderungen in dieser Datei und spec/fixtures/events.yml und spec/fixtures/groups.yml rückgängig machen